### ISSUE TITLE
Jarring transition between intro screen and "wizard" screen in launcher

### ISSUE THREAD

**Issue by edewit** [fabric8-launcher/ngx-launcher#89]
_From @bmicklea on February 27, 2018 23:49_

Problem:
The space creation and initial screen for the launcher are both black overlays that cover the OSIO top bar. Having gone through two screens similarly formatted the user believes that they should expect this style of screen as the configuration experience.

However, following the import or new project selection the user is dropped into a vertical "wizard" that is no longer an overlay and switches to main content on white background.

The effect is jarring and confuses the user because it seems to indicate that they're in a different part of the product and doing some other type of config (or perhaps not doing config at all?).

Possible solution:
It would be clearer to maintain consistency with:
- Overlay or non-overlay
- All black or black-nav with white-input area

_Copied from original issue: fabric8-launcher/launcher-planning#114_

**Comment by edewit** [comment]
_From @gastaldi on February 27, 2018 23:50_

I think this issue belongs to ngx-launcher?

**Comment by edewit** [comment]
_From @bmicklea on February 27, 2018 23:52_

I'm not sure @ALRubinger asked me to drop it here...

**Comment by edewit** [comment]
_From @gastaldi on February 27, 2018 23:56_

They seem to be very specific to the UI. 

Feel free to continue adding here, we can always move them to another repository later when triaging

**Comment by edewit** [comment]
_From @ALRubinger on February 28, 2018 0:34_

Yep, main triage here and we'll put it in the appropriate component repo as necessary.  Sending to Erik to figure who he'd like to have handle it and what to do. :)

**Comment by edewit** [comment]
_From @bmicklea on February 28, 2018 0:34_

If it's better to move them that's fine...I don't mind.

**Comment by edewit** [comment]
@dgutride @catrobson This is an issue of how the new wizard is integrated into OSIO do you have thoughts?

**Comment by catrobson** [comment]
Yes, this is an OSIO issue, but might also feel the same way when we use it in Launcher.  I'll take a look at this with the UX team this week.

**Comment by bmicklea** [comment]
@catrobson any update?

**Comment by catrobson** [comment]
@bmicklea no updates yet, we had to prioritize getting the final "status" page visual design finalized for the development team so we haven't wrapped back around to look at this issue yet.
